src/style.css

```diff
@@ -0,0 +1,21 @@
+.blue {
+  height: 100px;
+  background-color: blue;
+  width: 100%;
+}
+
+.green {
+  height: 100px;
```

Comment:
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.